### PR TITLE
Move og utils to a shared package

### PR DESCRIPTION
**Pull request by Hacksore**
Share the og utils to limit concurrent vercel builds

### CODE REVIEW

File: packages/og-utils/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@repo/og-utils",
+  "version": "0.0.0",
+  "private": true,
+  "exports": {
+    ".": "./src/index.ts"
+  },
+
+  "typesVersions": {
```
Comment (Hacksore) [inline packages/og-utils/package.json:9]: this makes importing work but IIRC there is a better way to do this now.

cc @anthonyshew

**Review by bautistaaa — APPROVED** [review]

File: packages/og-utils/package.json (hunk 1 above)
Comment (anthonyshew) [reply packages/og-utils/package.json:8]: You shouldn't need `typesVersions` at all. 🤔

Comment (Hacksore) [reply packages/og-utils/package.json]: removed it in https://github.com/typehero/typehero/pull/1545/commits/35b50b4d01830603f006a402ae0863b72acdba6a lets see 🙏

Comment (Hacksore) [reply packages/og-utils/package.json]: @anthonyshew i removed it 💀
image

Comment (Hacksore) [reply packages/og-utils/package.json:8]: we can revisit this but i reverted it so we can ship this and figure out later.

Comment (anthonyshew) [reply packages/og-utils/package.json:8]: Yeah, revisit. Should be able to do that just fine...

**Review by anthonyshew — APPROVED** [review]

**Pull request merged**